test/issue-2010/1.test.out.golden

```diff
@@ -1,3 +1,19 @@
+    #Not ( #Exists I . {
+      Some I
+    #Equals
+      f ( )
+    } )
+  #And
+    #Not ( {
+      None
+    #Equals
+      f ( )
+    } )
+  #And
+    <k>
+      f ( ) ~> .
+    </k>
```

Comment:
I'm not 100% certain of the behavior `K` is expected to have here. These are two behaviors I could think of, and I don't know which is expected:

1. Since `Some` and `None` are the only (visible) constructors for `MaybeInt`, this branch should be `\\bottom`.
2. Since `K` sorts are open, we do not assume that the only way to construct sort `MaybeInt` is with `Some` or with `None`, so this branch should exist.

Reply:
(1) is the correct behavior, and we rely on the SMT solver for this, so something is not being sent correctly to the solver.